weave-js/src/components/Panel2/PanelPlot/PanelPlot.tsx

```diff
@@ -281,6 +272,86 @@ const stringHashCode = (str: string) => {
   return hash;
 };
 
+const useConcreteConfig = (
```

Comment:
Love this - this pattern seems useful: given a config, some subset of the paths are nodes to be resolved. Here, that is defined using `LAZY_PATHS` (could this be just any config field with a node value... hmmm no not for things like column def). Anyhow, not in this PR, but it feels like a really powerful abstraction and close if we could just have any config field be lazy

Reply:
thanks. yeah the lazy_paths are needed because we dont want to evaluate some nodes (like select functions)